### ISSUE TITLE
Error with 2.5.0 and WordPress 5.8.1

### ISSUE THREAD

**Issue by stevejburge** [publishpress/PublishPress-Future#110]
Notice: Trying to access array offset on value of type bool in /wordpress/****/wp-content/plugins/post-expirator/post-expirator.php on line 271

**Comment by contactashish13** [comment]
@stevejburge I cannot reproduce this error. I cleared the database of all the settings and went directly to edit a post (with and without the Classic Editor active) but no luck. Can you please mention how to reproduce this and what the settings are?

**Comment by stevejburge** [comment]
@contactashish13 Is this info enough to go on? https://wordpress.org/support/topic/wordpress-5-8-1/

There's 2 reports there

**Comment by contactashish13** [comment]
@stevejburge I'm unable to reproduce this. I have tried with both Classic Editor switched on and off, with the Editor settings to both Show Gutenberg style box and Show Classic Editor style box and with a variety of post types. This could be happening with users who had certain old settings and if we can duplicate those settings we may be able to reproduce this.

Theoretically this can only happen when nothing is set for a specific post type and the user is editing the post but practically I cannot see the error. So I have added a fix for that potential use case - https://github.com/publishpress/PublishPress-Future/pull/119

PS - Can there be a way to ask users to export their plugin settings? Maybe introduce an option in the Diagnostics tab that exports all the settings? It would help replicate these niche issues.
